### PR TITLE
gnomeExtensions.no-title-bar: 9 -> 11, unbreak

### PR DESCRIPTION
**Pull request by amaxine**
###### Motivation for this change
Second attempt at #88679, since upstream tagged a release. I've been running this on my system for a while, it all works.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/desktops/gnome-3/extensions/no-title-bar/default.nix [hunk 1]
```diff
@@ -2,18 +2,16 @@
...
-  version = "9";
+  version = "11";
 
   src = fetchFromGitHub {
-    owner = "franglais125";
+    owner = "poehlerj";
     repo = "no-title-bar";
-    rev = "v${version}";
-    sha256 = "02zm61fg40r005fn2rvgrbsz2hbcsmp2hkhyilqbmpilw35y0nbq";
+    rev = "V_11";
```
Comment (drewrisinger) [inline pkgs/desktops/gnome-3/extensions/no-title-bar/default.nix:10]:
```suggestion
    rev = "V_${version}";
```

**Review by drewrisinger — CHANGES_REQUESTED** [review]
* [x] Diff LGTM minus minor comment
* [x] Commits should probably be squashed
* [x] Builds fine via ``nix-review`` on x86_64-linux. No further testing done (now sure how, on headless machine).

```
https://github.com/NixOS/nixpkgs/pull/106828
1 package built:
gnomeExtensions.no-title-bar
```

File: pkgs/desktops/gnome-3/extensions/no-title-bar/default.nix (hunk 1 above)
Comment (maxeaubrey) [reply pkgs/desktops/gnome-3/extensions/no-title-bar/default.nix]: Fixed. Thanks for catching that, despite doing so in my personal repo somehow I didn't here? Oops :sweat_smile:

**Review by drewrisinger — APPROVED** [review]
* [x] Diff LGTM
* [x] Commits LGTM
* [x] Builds via ``nix-review``:
* [ ] No further testing done by me

```
https://github.com/NixOS/nixpkgs/pull/106828
1 package built:
gnomeExtensions.no-title-bar
```

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 106828` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>gnomeExtensions.no-title-bar</li>
</ul>
</details>

**Comment by maxeaubrey** [comment]
Is there anything I can do to get this merged? Would like to see this in master before 21.05 and there's another release of this extension :sweat_smile:

**Comment by buckley310** [comment]
I'm currently testing with this patch applied to unstable. Works just fine. I may end up keeping it :)

**Pull request merged**